its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -84,7 +84,7 @@ public static void verifyGuiTestOnlyProjectAnalysisWarning(Orchestrator orchestr
       "Read more about how the SonarScanner for .NET detects test projects: https://github.com/SonarSource/sonar-scanner-msbuild/wiki/Analysis-of-product-projects-vs.-test-projects");
   }
 
-  public static Ce.Task getAnalysisWarningsTask(Orchestrator orchestrator, BuildResult buildResult) {
+  private static Ce.Task getAnalysisWarningsTask(Orchestrator orchestrator, BuildResult buildResult) {
```

Comment:
I don't want to change the order based on visibility in this file, there are many methods that don't respect it and it would make this PR messy